.github/workflows/python-deploy.yml

```
@@ -62,13 +62,13 @@ jobs:
           (. .venv/install/${{ env.BIN_DIR }}/activate && python -m pip install --no-index --find-links=partcad-cli/dist partcad-cli && deactivate)
       # Upload artifact from the selected OS/Python version combination only
       - name: Upload "partcad"
-        if: "matrix.os == 'ubuntu-latest' && matrix.python-version == '3.11'"
+        if: "matrix.os == 'ubuntu-24.04' && matrix.python-version == '3.11'"
         uses: actions/upload-artifact@v3
```

actionlint (1.7.4)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)